service/src/main/resources/db/changesets/20221011_unique_managed_application.yaml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: unique_managed_application
+      author: mtalbott
+      changes:
+        - addUniqueConstraint:
```

Comment:
Are there any existing profiles in dev that will violate this constraint and cause the migration to fail when applied?

Reply:
Should it just be unique between `subscription_id` and `managd_resource_group_id`?

Reply:
> Are there any existing profiles in dev that will violate this constraint and cause the migration to fail when applied?

Hadn't checked, but yes there are a number of existing profiles that will violate this constraint. I believe I can get around that by making the constraint deferrable, I'll play around with that.

Reply:
> Should it just be unique between `subscription_id` and `managd_resource_group_id`?

I went with all three because they represent the full managed app coordinates, but I just checked and Azure subscription IDs are globally unique, so yes we can make this unique just between the subscription and managed resource group id.

Reply:
Many of the problematic profiles are probably mine. If it becomes a problem, I am fine deleting mine.

Reply:
constraint deferral did not do the trick, looks like the only option is to delete the offending billing profiles. Fortunately, all of the offending profiles have been created by our team. I'll post the list in Slack and make sure everyone's okay with me deleting duplicates